src/Microsoft.AspNetCore.Razor.Design/Internal/AssemblyTagHelperDescriptorResolver.cs

```diff
@@ -61,6 +52,47 @@ public IEnumerable<TagHelperDescriptor> Resolve(string assemblyName, ErrorSink e
                         DesignResources.InvalidProtocolValue,
                         typeof(TagHelperDescriptor).FullName, ProtocolVersion));
             }
+
+            if (!_loadedAssembly)
+            {
+                LoadMvcAssembly();
+            }
+
+            if (_hasMvcAssembly)
```

Comment:
Can we check if `_populateMethod` is not null instead of having a separate variable?